cosmo_tester/resources/blueprints/active-directory/scripts/wait_for_active_directory_port.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+HOST_IP=$public_ip
+PORT=$port
+TIMEOUT_MIN="$(($timeout_minutes * 60))"
+function wait_for_port
+{
+    /bin/netcat -z -w$3 $1 $2
+    if [ $? -ne 0 ] && $fail_on_timeout; then
```

Comment:
1. The netcat command does not raise an error and there for will not terminate the script with exit code 1.
2. The error message is fine the way it is.